applications/endoscopy_tool_tracking_distributed/cpp/metadata.json

```diff
@@ -32,7 +32,7 @@
 			]
 		},
 		"run": {
-			"command": "HOLOSCAN_INPUT_PATH=<holohub_data_dir>/endoscopy <holohub_app_bin>/endoscopy_tool_tracking_distributed",
+			"command": "<holohub_app_bin>/endoscopy_tool_tracking_distributed",
```

Comment:
Does the user need to set the environment variable (`HOLOSCAN_INPUT_PATH`) before executing this command on the host system (without a Docker container)? How will the user know about the requirement for the environment variable if it is not specified in this command?

Reply:
We should add/use the `--data` from the command line. This is what we have for most of the applications.
See https://github.com/nvidia-holoscan/holohub/blob/main/applications/endoscopy_tool_tracking/cpp/metadata.json#L35 for reference

Reply:
Okay. Gigon confirms that arguments still work; here's an excerpt from the `application.hpp` source code:

>    * This constructor parses the command line for flags that are recognized by App Driver/Worker,
>   * and removes all recognized flags so users can use the remaining flags for their own purposes.

Let me update the app to use `--data`

Reply:
Awesome - thanks for looking into this!